.all-contributorsrc

```diff
@@ -18,6 +18,33 @@
         "infra",
         "test"
       ]
+    },
+    {
+      "login": "gyandeeps",
+      "name": "Gyandeep Singh",
+      "avatar_url": "https://avatars.githubusercontent.com/u/5554486?v=3",
+      "profile": "http://gyandeeps.com",
+      "contributions": [
+        "review"
+      ]
+    },
+    {
+      "login": "exdeniz",
+      "name": "Igor Pnev",
+      "avatar_url": "https://avatars.githubusercontent.com/u/682584?v=3",
+      "profile": "https://github.com/exdeniz",
+      "contributions": [
+        "bug"
+      ]
+    },
+    {
+      "login": "demoneaux",
```

Comment:
@ demoneaux, are you ok with me adding you to the contributors list? See [this](https://github.com/kentcdodds/prettier-eslint/blob/e8518fc647c6374ceeddea7452f0cb23a25eece5/README.md#contributors) for what it will look like.